### PR TITLE
Task to check outdated blogposts on demand

### PR DESCRIPTION
**Pull request by arielj**
This script doesn't require an event on the blog that uses the app.

It will loop over all the installations, authenticate, fetch all the articles and create branches, commits, pull requests and issues for the articles last updated (exactly) 3 months ago (notice 2020-05-31 - 3 months in ruby returns 2020-02-29 so some dates may be checked more than one time, but an article can only have one branch at a time so it's not that bad).

Right now it's skipping all installations other than ombulabs and fastruby for testing purposes. Once it gets merged I can work on configuring Scheduler addon on heroku to run this script once a day.

### CODE REVIEW

**Comment by ABizzinotto** [comment]
@lubc could you review this today? Thanks!

File: check_outdated_posts.rb [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    end
+  end
+
+  def authenticate_installation(installation)
+    puts "Authenticating: #{installation[:account][:login]}"
+    # clear token
+    client.access_token = nil
+
+    # get token for current installation
+    token = client.create_app_installation_access_token(installation.id)
```
Comment (lubc) [inline check_outdated_posts.rb:72]: @arielj Maybe do `token = client.create_app_installation_access_token(installation.id).token` so you don't have `token.token` below?

File: tests.rb [hunk 2]
```diff
@@ -0,0 +1,139 @@
+require_relative './article_handler.rb'
+require_relative './repository_handler.rb'
+require "minitest/autorun"
+
+describe ArticleHandler do
```
Comment (lubc) [inline tests.rb:5]: @arielj Maybe it would be cleaner to break this test into two files. One for `ArticleHandler` and one for `RepositoryHandler`?

**Comment by arielj** [comment]
> @arielj Great work on this! 👏
> 
> I guess now we can get rid of the server.rb and all the Sinatra stuff since we won't respond to webhooks anymore, right?

Yeah, that's the idea I think, but I didn't just remove that because there are people using blogcop, I wanted to first use the crontab internally and when we are sure it's working on production then we can remove the sinatra app.

**Pull request merged**